test/configCases/plugins/provide-plugin/webpack.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
 			es2015_name: ["./harmony", "default"],
 			es2015_alias: ["./harmony", "alias"],
 			es2015_year: ["./harmony", "year"],
-			"this.aaa": "./aaa"
+			"this.aaa": "./aaa",
+			esm: "./fff.mjs"
```

Comment:
change this to `esm: "fail"`

import `fff.mjs` from `index.js`.

Use `esm` in `fff.mjs`. It should not be replaced. i. e.

``` js
export function() {
  return esm;
}
```

---

Currently you are not testing your change.